templates/config.json

```
@@ -598,6 +641,13 @@
       "provider": "JupiterOne",
       "summary": "Employees and contractors receive training on the organization's security policies and procedures.",
       "guidance": "You will need to ensure all your employees are properly trained on the security policies and procedures. You may leverage a full featured learning management system (LMS) to develop, distribute, and track this training. JupiterOne also supports a simple process that allows your employees to access your security policies and procedures, and capture their acknowledgement.",
+      "instructionsSummary": "Confirm/tailor training program: retention duration, compliance frameworks, inactivity logout duration, etc.",
+      "instructions": [
```

Remove the `instructionsSummary` and move it to the first string in the array:

```suggestion
      "instructions": [
        "Confirm/tailor training program: retention duration, compliance frameworks, inactivity logout duration, etc.",
```